code/modules/mob/living/simple_animal/bot/griefsky.dm

```diff
@@ -0,0 +1,403 @@
+/mob/living/simple_animal/bot/griefsky //This bot is powerful. If you managed to get 4 eswords somehow, you deserve this horror.
+	name = "\improper General Griefsky"
+	desc = "Is that a secbot with four eswords in its arms...?"
+	icon = 'icons/obj/aibots.dmi'
+	icon_state = "griefsky0"
+	density = 0
+	anchored = 0
+	health = 150
+	maxHealth = 150
+	damage_coeff = list(BRUTE = 0.5, BURN = 0.7, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	pass_flags = PASSMOB
+	radio_channel = "Security" //Security channel
+	bot_type = SEC_BOT
+	bot_filter = RADIO_SECBOT
+	model = "Securitron"
+	bot_purpose = "seek out criminals, end their miserable lifes"
+	bot_core_type = /obj/machinery/bot_core/secbot
+	window_id = "autosec"
+	window_name = "Automatic Security Unit v9.0"
+	path_image_color = "#FF0000"
+	data_hud_type = DATA_HUD_SECURITY_ADVANCED
+	allow_pai = 0
+	var/weapon //edagger or esword 
+	weapon = /obj/item/melee/energy/sword
+	
+	var/spin_icon = "griefsky-c" // griefsky and griefsky junior have dif icons
+	var/dmg = 30 //esword dmg
+	var/block_chance_melee = 50
+	var/block_chance_ranged = 90
+	var/stun_chance = 50
+	var/spam_flag = 0
+	var/base_icon = "griefsky"
+	var/mob/living/carbon/target
+	var/oldtarget_name
+	var/threatlevel = 0
+	var/target_lastloc //Loc of target when arrested.
+	var/last_found //There's a delay
+	var/declare_arrests = 1 //When making an arrest, should it notify everyone on the security channel?
+	var/idcheck = 0 //If true, arrest people with no IDs
+	var/weaponscheck = 0 //If true, arrest people for weapons if they lack access
+	var/check_records = 1 //Does it check security records?
+	var/arrest_type = 0 //If true, don't handcuff
+
+	
+/mob/living/simple_animal/bot/griefsky/proc/spam_flag_false() //used for addtimer to not spam comms
+	spam_flag = 0
+
+/mob/living/simple_animal/bot/griefsky/New()
+	..()
+	icon_state = "[base_icon][on]"
+	spawn(3)
+		var/datum/job/detective/J = new/datum/job/detective
+		access_card.access += J.get_access()
+		prev_access = access_card.access
+
+	//SECHUD
+	var/datum/atom_hud/secsensor = huds[DATA_HUD_SECURITY_ADVANCED]
+	secsensor.add_hud_to(src)
+	permanent_huds |= secsensor
+
+/mob/living/simple_animal/bot/griefsky/jgriefsky  // cheaper griefsky less damage
+	name = "Griefsky apprentice"
+	desc = "Is that a secbot with four energy daggers in its arms...?"
+	dmg = 18 // e dagger
+	block_chance_melee = 30
+	block_chance_ranged = 50
+	spin_icon = "griefskyj-c"
+	window_name = "Automatic Security Unit v7.0"
+	health = 100
+	maxHealth = 100
+	weapon = /obj/item/pen/edagger
+	
+
+/mob/living/simple_animal/bot/griefsky/griefsky/toy  //A toy version of general griefsky!
+	name = "Genewul Bweepskee"
```

Comment:
I would think the toy version name would be similar to the new griefy one but I don't know. I would have done Gweafskee I think